connection-service/new-app-gateway/azuredeploy.json

```diff
@@ -71,11 +71,11 @@
           "description": "'true' if a Public IP address should be created, otherwise 'false'."
       }
     },
-    "enableRadiusMfa": {
+    "disableExternalUserInterface": {
       "type": "bool",
       "defaultValue": false,
       "metadata": {
-        "description": "Indicates weather or not MFA is enabled"
+        "description": "Indicates weather or not to make UI extnerally accessible"
```

Comment:
whether

Reply:
"If true, disables external access to the management interface." ?

Reply:
fixed